tests/test_frontdoor.py

```diff
@@ -1465,6 +1481,13 @@ def test_on_check_answers_page_changing_to_social_housing_hides_park_home_questi
     page = page.click(contains="Back")
     assert page.has_one("h1:contains('Check your answers')")
 
+    if park_home:
+        _assert_change_button_is_not_hidden(page, "park-home")
+        _assert_change_button_is_not_hidden(page, "park-home-main-residence")
+    else:
+        _assert_change_button_is_not_hidden(page, "park-home")
+        _assert_change_button_is_hidden(page, "park-home-main-residence")
```

Comment:
We could pull the first assertion outside the if/else, since it's the same in both places.

I kind of think the `park-home-main-residence` assertion doesn't belong here: it's sort of orthogonal to what the test is testing. It's really testing the logic around hiding that bit depending on the park home answer, which has its own test (I think? Should do!). Unit tests should really only test one thing, and shouldn't overlap: that means if a test fails then you know immediately what's gone wrong, and don't have to spend time debugging the test itself

(I'd maybe even drop the `park_home` parameter from the test now I look at it, and just cover one case arbitrarily)

Reply:
> (I think? Should do!).

I've split the test into two to ensure we're not losing any coverage :)